addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -58,9 +56,9 @@
 
                             <div class="d-flex align-items-center">
                                 <div class="progress flex-grow-1" style="height:0.5em">
-                                    <div class="progress-bar bg-primary" t-att-style="'width: '+ str(channel_completion)+'%'"/>
+                                    <div class="progress-bar bg-primary" t-att-style="'width: '+ str(course.completion)+'%'"/>
```

Comment:
Could use t-attf-style to make this a bit more readable

Reply:
Indeed, but it's a little late now (I just noticed your comment).

Reply:
You should be more readable.